src/models.rs

```diff
@@ -0,0 +1,266 @@
+use std::collections::HashMap;
+
+/// # Model definitions, by bios product name
+/// - flash: products that will automatically install updates
+/// - ec: product has embedded controller firmware
+/// - ec2: product has second embedded controller firmware
+/// - me: product has management engine firmware
+pub struct ModelSupport {
+    pub flash: bool,
+    pub ec: bool,
+    pub ec2: bool,
+    pub me: bool,
+}
+
+pub fn models() -> HashMap<&'static str, ModelSupport> {
+    let mut models = HashMap::new();
+
+    models.insert("addw1", ModelSupport {
```

Comment:
The ModelSupport data is no longer necessary. This could just return a &'static [&'static str] of models

Reply:
Revised commit here: https://github.com/pop-os/system76-firmware/pull/15/commits/4e61880afaa3e6d59a5cb72fa9655b0847391d4d